index.js

```diff
@@ -1,6 +1,11 @@
 module.exports = {
   'extends': 'standard',
   'rules': {
+    'array-bracket-spacing': [
+      1,
+      'always'
+    ],
+    'no-multi-spaces': 1,
```

Comment:
Is this an unrelated rule?

Reply:
i.e. does it have ripple effects beyond arrays

Reply:
It does go beyond arrays, yes. https://eslint.org/docs/rules/no-multi-spaces

It's enabled already in the `standard` config, but since we're setting the other, we should make sure this is set as well. `array-bracket-spacing` can only be set to "never" or "always", where "always" means "one or more." We want only one, so by setting this rule we're enforcing a single space.